Makefile

```diff
@@ -50,70 +49,61 @@ autoloads:
 
 # Create .tgz and .zip files only
 # GNUTAR=gtar make tarballs
-tarballs: $(ESSDIR)
+.PHONY: rel-tarballs
+rel-tarballs: rel-staging
 	@echo "**********************************************************"
 	@echo "** Making distribution of ESS for (pre)release $(ESSVERSION) from $(ESSDIR)/"
-	@echo "** Making pdf and html documentation"
-	cd $(ESSDIR)/doc/ ; $(MAKE) pdf
-	cd $(ESSDIR)/doc/ ; $(MAKE) html
-	@echo "** Creating .tgz file **"
 	test -f $(ESSDIR).tgz && rm -rf $(ESSDIR).tgz || true
+	grep -E 'defvar ess-(version|revision)' $(ESSDIR)/lisp/ess-custom.el
```

Comment:
What does this grep do?

Reply:
You might have to ask [mmaechler](https://github.com/emacs-ess/ESS/blob/master/Makefile#L125) but I think #753 obviates it.  Removing with extreme prejudice.

Reply:
versions are not longer in ess-custom but in ess.el and that grep is almost surely not needed.